base_layer/wallet/src/output_manager_service/tasks/txo_validation_task.rs

```diff
@@ -409,18 +418,36 @@ where
 
         let mut returned_outputs = HashMap::new();
         for output_proto in &batch_response.responses {
-            returned_outputs.insert(output_proto.output_hash.clone(), output_proto);
+            match FixedHash::try_from(output_proto.output_hash.clone()) {
+                Ok(v) => {
+                    returned_outputs.insert(v, output_proto);
+                },
+                Err(_) => {
+                    debug!(
```

Comment:
I think this should be a warn, so that it's apparent that a base node is giving bad data